packages/react-menu/src/components/Menu/Menu.types.ts

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { ComponentProps, ObjectShorthandProps, ShorthandProps } from '@fluentui/react-utilities';
+import { MenuListProps } from '../MenuList/index';
+
+/**
+ * Extends and drills down Menulist props to simplify API
+ * {@docCategory Menu }
+ */
+export interface MenuProps extends ComponentProps, React.HTMLAttributes<HTMLElement>, MenuListProps {
+  /**
+   * Whether the popup is open
+   */
+  open?: boolean;
+
+  /**
+   * Whether the popup is open by default
+   */
+  defaultOpen?: boolean;
+
+  /**
+   * Callback to open/close the popup
+   */
+  setOpen?: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
```suggestion
```

Should be this really in `MenuProps`? 🤔

Reply:
The menu popup should be controlled. I don't really see much sense in putting it in the trigger or menulist ?

Reply:
or would we rather be following the `onOpenChanged` pattern ?

Reply:
oh wait yeah there's no point in this.... brain froze for a sec :)